Linux/Car_Client/chronos.cpp

```diff
@@ -31,8 +30,6 @@ Chronos::Chronos(QObject *parent) : QObject(parent)
             this, SLOT(processOstm(chronos_ostm)));
     connect(mChronos, SIGNAL(strtRx(chronos_strt)),
             this, SLOT(processStrt(chronos_strt)));
-    connect(mChronos, SIGNAL(oproRx(chronos_opro)),
-            this, SLOT(processOpro(chronos_opro)));
```

Comment:
Was this not used, or why is it being deleted?

Reply:
It is duplicated, check the code on line 27